src/components/Todo/Todo.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Todo.css';
+
+const Todo = ({ todo }) => (
+  <li className="TodoList__Todo Todo" hidden={todo.completed}>
+    {todo.title}
```

Comment:
Тут варто було б ще виводити id користувача

```suggestion
    {todo.title} {todo.userId}
```